app.css

```diff
@@ -87,15 +87,11 @@ canvas {
 }
 
 .instructions{
```

Comment:
I like it! maybe we could tweak the colors a little to match with the scheme of the game window...I think we should add a shadow to the game field too.